src/ontology/mondo-edit.obo

```diff
@@ -553368,6 +553368,7 @@ name: variable-age onset focal epilepsy syndrome
 def: "An epilepsy syndrome characterized by focal seizures where age at seizure onset varies." [https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html]
 synonym: "VAOFAS" EXACT [https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html]
 is_a: MONDO:0005384 {source="https://orcid.org/0000-0002-4142-7153"} ! focal epilepsy
+is_a: MONDO:0015650 ! epilepsy syndrome
```

Comment:
@Daniel-Olson all of the superclass assertions in Mondo need to have a source
please add this as the source: PMID: 40264351